staff/alex-barba/gamma-template/contact.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style/style.css">
+    <title>Contact Us</title>
+</head>
+<body>
+    <header class="menu">
+        <h1 class="menu__title">Gamma</h1>
+        <ul class="nav">
+            <li>Home</li>
+            <li>About</li>
+            <li>Contact</li>
+        </ul>
+        <img class="menu__button" src="./style/menu.svg" alt="">
+    </header>
+    <main class="main">
```

Comment:
If you are not using the class main to give style, you don't need to add it

Reply:
Yes, it makes no sense...